src/test/java/io/vertx/core/http/HttpTestBase.java

```diff
@@ -34,7 +34,7 @@
 public class HttpTestBase extends VertxTestBase {
 
   public static final String DEFAULT_HTTP_HOST = "localhost";
-  public static final int DEFAULT_HTTP_PORT = 8080;
+  public static final int DEFAULT_HTTP_PORT = Integer.parseInt(System.getProperty("port", "8080"));
```

Comment:
instead name it `vertx.httpPort` , we already have a few of them like `vertx.useNativeTransport`, etc...